tests/test_utils/graphs.py

```diff
@@ -111,7 +111,7 @@ def example_hin_1_nx(feature_name=None, for_nodes=None, feature_sizes=None):
 
 
 def example_hin_1(
-    feature_sizes=None, is_directed=False, self_loop=False
+    feature_sizes=None, is_directed=False, self_loop=False, reverse_order=False
```

Comment:
What's this new flag for?

Reply:
ah that flag reverses the order of the node ids so `node_id` != `node_iloc`

Reply:
Ah, I see.